autosklearn/automl.py

```diff
@@ -114,9 +114,11 @@ def send_warnings_to_log(
 class AutoML(BaseEstimator):
 
     def __init__(self,
-                 backend: Backend,
                  time_left_for_this_task,
                  per_run_time_limit,
+                 temporary_directory: Optional[str] = None,
+                 output_directory: Optional[str] = None,
```

Comment:
The output directory is no longer necessary, right?

Reply:
Thanks, I clearly missed this and it was an artifact from the rebase